pkg/subctl/cmd/gather/logs.go

```diff
@@ -48,9 +48,10 @@ func gatherPodLogsByContainer(podLabelSelector, container string, info Info) {
 		podLogOptions := corev1.PodLogOptions{}
 		podLogOptions.Container = container
 		for i := range pods.Items {
+			info.Summary.PodLog.LogFileName = nil
```

Comment:
It seems like you're using `PodLog` essentially as a global when it really should be local. You should have have `outputPodLogs` return the `PodLog` info.

Reply:
This was done in https://github.com/submariner-io/submariner-operator/pull/1420. It seems you've partially incorporated those changes here. That PR also includes this one. Is the intention to merge that one?

Reply:
I’ve merged the changes into this PR.